packages/kit/src/components/OneKeyAuth/supabase/README.md

```
@@ -0,0 +1,177 @@
+  - https://supabase.com/dashboard/project/wtspqckturkzhstyjabx/auth/protection
+  - https://supabase.com/docs/guides/auth/rate-limits
+  - https://supabase.com/dashboard/project/wtspqckturkzhstyjabx/auth/rate-limits
+  - Supabase Auth Hooks
+- 客户端伪造钓鱼攻击 TODO
+- 域名白名单 TODO
+- email 登录 redirect 域名约束
+- 
+
+# Errors
+
+https://supabase.com/docs/guides/auth/debugging/error-codes
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)